extension/content_script.js

```diff
@@ -88,6 +88,7 @@ function setupSentenceClickListeners() {
 		doc.getSentenceEls(sentenceId).on("click", function(e) {          
 			highlight(sentenceId);
 			tracker.pointToSentence(sentenceId);
+			scrollToTracker();
```

Comment:
I thought when u click it would be nice to autoscroll to view

Reply:
V nice indeed